tails_files/install.sh

```diff
@@ -74,7 +74,7 @@ Host mon
 EOL
     chown amnesia:amnesia $INSTALL_DIR/ssh_config
     chmod 600 $INSTALL_DIR/ssh_config
-    cp -p $INSTALL_DIR/ssh_config $HOMEDIR/.ssh/config
+    cp -pf $INSTALL_DIR/ssh_config $HOMEDIR/.ssh/config
```

Comment:
Does the ssh config need to be copied to the `$INSTALL_DIR`? AFAICT it's already persisted because the [SSH persistence](https://tails.boum.org/doc/first_steps/persistence/configure/index.en.html#index3h2) includes `~/.ssh/config/`.